Makefile

```diff
@@ -0,0 +1,7 @@
+# Main makefile for the testing framework. It should be executed by the actual project containing the tests
+.PHONY: init
+
+init:
+	rm -rf shunit2
+	git clone --depth 1 https://github.com/kward/shunit2
```

Comment:
🐛 version is not pinned here. In such case it's a matter of time till it explodes

Reply:
So far, the released tags do not contains all the macros and functions that master branch does (assertContains, e.g.). I'll add a TODO comment and once a new version containing all the functionalyty is released we will change and clone using --branch.

Reply:
ok!. I'm not fan of using a commit to be sincere, but since we have to keep the TODO, I'll make that simple change before merging